Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:14.04
+
+ADD ./WcaOnRails /app/WcaOnRails
```

Comment:
That's what `VOLUME` is for (or `volumes:` in docker-compose). `ADD` is basically a more advanced version of `COPY`, see https://docs.docker.com/engine/reference/builder/#add.

You probably won't notice the difference unless you stop the container and run `docker-compose build`.